evm/vm/forks/eip170/constants.py

```diff
@@ -0,0 +1,2 @@
+# https://github.com/ethereum/EIPs/issues/170
+EIP170_CODE_SIZE_LIMIT = 24577
```

Comment:
instead of `forks.eip160` this should be in `forks.byzantium` (since this is actually part of byzantium's fork rules)

Reply:
The above comment applies to this change as well.